packages/kununu-utils/newrelic/getConfig/getConfig.spec.js

```diff
@@ -1,8 +1,8 @@
-const isDisabled = require('../isDisabled');
+const isEnabled = require('../isEnabled');
```

Comment:
Maybe we could make this more descriptive? What exactly **isEnabled?**

Reply:
well it's in the newrelic folder, so I think it's descriptive enough